### ISSUE TITLE
Use PostCSS 5.0

### ISSUE THREAD

**Issue by ai** [postcss/postcss-import#75]
Seems like `postcss.parse` broke a AST in PostCSS 5.0 (because your plugin depends on PostCSS 4.x and uses 4.x parser, which generates 4.x nodes).

Can you update PostCSS in dependencies?

/cc @MoOx

**Comment by ai** [comment]
We add special hook fo 4.x plugin in PostCSS 5.0, but I found a [bad place](https://github.com/postcss/postcss-import/blob/master/index.js#L457) in `postcss-import`:

```js
https://github.com/postcss/postcss-import/blob/master/index.js#L457
```

By the many reasons, changing `nodes` directly is not a good way :).

**Comment by MoOx** [comment]
I used a low level array manipulation because of all the bug the cloning methods were introducing.

**Comment by MoOx** [comment]
Closed by 062693944b992a3cf92f9c3ae3e3e6633df59f1d
